Parsers/ASimUserManagement/Parsers/ASimUserManagementCiscoISE.yaml

```diff
@@ -0,0 +1,60 @@
inactivity","60461","Success","UserDisabled","UserModified","Informational","INFO","Account disabled due to user level date expiry","60462","Success","UserDisabled","UserModified","Informational","INFO","Account disabled due to global level date expiry","60463","Success","UserDisabled","UserModified","Informational","INFO","Account disabled due to global level days expiry","10013","Success","UserModified","UserModified","Informational","INFO","Admin account set as 'never disabled'","10014","Success","UserModified","UserModified","Informational","INFO","Admin account set to change password on next login","5415","Failure","PasswordChanged","UserModified","Low","NOTICE","Change password failed","86002","Success","UserDisabled","UserModified","Informational","INFO","Sponsor has suspended a guest user account","86003","Success","UserEnabled","UserModified","Informational","INFO","Sponsor has enabled a guest user account","86004","Success","PasswordChanged","UserModified","Informational","INFO","Guest user has changed the password","86006","Success","UserCreated","UserCreated","Informational","INFO","Guest user account is created","86007","Success","UserModified","UserModified","Informational","INFO","Guest user account is updated","86008","Success","UserDeleted","UserModified","Informational","INFO","Guest user account is deleted","86015","Failure","PasswordChanged","UserModified","Low","INFO","Invalid Password Change","24059","Failure","PasswordChanged","UserModified","Low","ERROR","User password change ended with an error","24064","Failure","PasswordChanged","UserModified","Low","WARN","The user doesn't have sufficient rights to change password","24065","Failure","PasswordChanged","UserModified","Low","WARN","The new password does not conform to LDAP password policy","24066","Success","PasswordChanged","UserModified","Informational","INFO","User password change succeeded","24205","Failure","PasswordChanged","UserModified","Low","ERROR","Could not change password to new password","24206","Success","UserDisabled","UserModified","Informational","INFO","User disabled","24347","Success","UserDisabled","UserModified","Informational","ERROR","Account disabled","24348","Success","UserLocked","UserModified","Informational","ERROR","Account locked","24370","Success","UserDisabled","UserModified","Informational","ERROR","User credentials have been revoked.","24425","Success","PasswordChanged","UserModified","Informational","INFO","User change password against Active Directory succeeded","24426","Failure","PasswordChanged","UserModified","Low","ERROR","User change password against Active Directory failed","24455","Failure","PasswordChanged","UserModified","Low","ERROR","Change password against Active Directory failed because of a timeout error","33108","Success","PasswordReset","UserModified","Informational","INFO","Reset admin password to its default value","5204","Success","PasswordChanged","UserModified","Informational","NOTICE","Change password succeeded"];
```

Comment:
can we map EventResultDetails too?
https://learn.microsoft.com/en-us/azure/sentinel/normalization-schema-user-management#common-fields-with-specific-guidelines

Reply:
EventResultDetails added to lookup table